jwst/outlier_detection/outlier_detection.py

```diff
@@ -17,6 +17,7 @@
 from jwst.resample import resample
 from jwst.resample.resample_utils import build_driz_weight, calc_gwcs_pixmap
 from jwst.stpipe import Step
+from jwst.resample.resample import _compute_image_pixel_area
```

Comment:
Since this function is now needed outside of the resample module, I think we should probably rename it without the underscore.